modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -5,11 +5,7 @@
 [id="nw-ovn-kubernetes-limitations_{context}"]
 = OVN-Kubernetes limitations
 
-The OVN-Kubernetes Container Network Interface (CNI) cluster network provider has the following limitations:
-
-* OVN-Kubernetes does not support setting the external traffic policy or internal traffic policy for a Kubernetes service to `local`.
```

Comment:
@mikemckiernan: We still do not support Internal Traffic Policy. So please keep this bullet point intact but remove "external traffic policy" from it.